app/models.py

```diff
@@ -1431,6 +1453,17 @@ def validates_data(self, key, data):
 
         return data
 
+    @validates('awarded_at')
+    def validates_awarded_at(self, key, awarded_at):
+        if self.status == "awarded":
+            raise ValidationError('Brief response award cannot be changed as the brief has already been awarded.')
+        if awarded_at:
```

Comment:
Moving this case to the top would mean that we could potentially un-award a BriefResponse by setting the `awarded_at` datestamp to None, which we're currently disallowing. Should we be this strict? (My gut says no...)